pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -205,6 +222,28 @@ func (s *Server) watchRootCertAndGenKeyCert(stop <-chan struct{}) {
 	}
 }
 
+func (s *Server) watchDNSCertForK8sCA(stop <-chan struct{},
+	defaultCACertPath string,
+	signerName string,
+	approveCsr bool,
```

Comment:
nit: i can see all the caller set it true, so can remove it

Reply:
this value is set false in some scenarios in non-OSS code. Want to keep it to minimize diff.

Reply:
Resolving this comment in the favour of Shankar's comment, Please feel free to open the discussion again if changes are needed.